roles/runtime/tasks/initialize_base.yml

```diff
@@ -148,7 +148,8 @@
         overlay_network: "{{ __ml_config.network | default(include.network) | default({}) }}"
         base_instance_group: "{{ run__ml_k8s_request_base }}"
         config:
-          name: "{{ __ml_config.name | default([run__namespace, __ml_config.suffix | default(include.suffix) | default(run__ml_suffix)] | join('-')) }}"
+          name: "{{ __ml_config.name | default([run__namespace_cdp, __ml_config.suffix | default(include.suffix) | default(run__ml_suffix)] | join('-')) }}"
+          nfs: "{{ __ml_config.nfs | default(run__azure_nfs_mount | default(omit)) }}"
```

Comment:
Why have the default(omit)? Wouldn't it be better to fail here than get all the way to the cloud module that tries to create the CML workspace?

Reply:
because I am being overly clever by reusing the same preparation task for AWS and Azure, so if you are doing an AWS preparation and thus do not supply the NFS information, they configuration item will be omitted, and if you are doing Azure and it is missing you'll get the CLI failure. We could add the explicit requirement to cdpy to prevent it from submitting to the CLI tho, which would be nice.